gc/stats/GlobalGCStats.hpp

```diff
@@ -77,6 +77,16 @@ class MM_GlobalGCStats {
 		finalizableCount = 0;
 	};
 
+	/**
+	 * Get the total stall time
+	 * @return the time in hi-res ticks ???
```

Comment:
`???` should be removed  in the comment

Reply:
Previous commit failed after adding indent, which does not make sense to me. So I add `???` to restart the testing.  I will remove these  `???`.